pkg/repstream/streampb/empty.go

```diff
@@ -129,23 +130,43 @@ type DebugLogicalConsumerStatus struct {
 	}
 }
 
+const (
+	ProcessingState = "processing-other"
```

Comment:
These look wide; think about the table we're trying to consume and how hard it is to read if it wraps to multiple lines per row. I'd do `wait`, `flush`, and if you really want a non-flush other state, `other`

Reply:
I'd also move these strings to crdb_internal; the only thing we need in this struct is a iota byte

Reply:
shortened the names. Created a string method in streampb to be called in crdb_internal.